core/whats-new/whats-new-in-21/cs/brotli.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.IO;
+using System.IO.Compression;
+
+public static class BrotliExample
+{
+    // <Snippet1>
+    public static Stream DecompressWithBrotli(Stream toDecompress)   
+    {
+        MemoryStream decompressedStream = new MemoryStream();
+        using (BrotliStream decompressionStream = new BrotliStream(toDecompress, CompressionMode.Decompress))
+        {
+            decompressionStream.CopyTo(decompressedStream);
+        }
+        return decompressedStream;
```

Comment:
Shouldn't you reset the `Position` of the `decompressedStream` to zero just before the `return`, so that caller can directly read the returned `Stream`?